libraries/persistent_store/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 Google LLC
```

Comment:
Could technically be 2020 (also in other files), if those didn't exist on GitHub before?

Reply:
Those files as such did not exist (I don't think I started working on this before 2020 but maybe I did at least on the design), but they still share some similarity with the existing ones (like high level structure but also more details for `storage.rs` and `buffer.rs`). I think we can also write `2019-2020`. Not sure what to do here.

Reply:
Then `2019-2020` would be better.

Reply:
Done with 2019-2020.